CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,41 @@
+## Introduction
+
+WIRIS is dedicated to provide a good environment so that everybody can have a good experience. It's not tolerated any type of harassment or violation of physical boundaries of partifipants. We ask you to be considetare to others and behave respectfull and professionally. 
```

Comment:
provide -> providing
"It's not tolerated any type of harassment" sounds weird to me. Maybe "No type of harassment is tolerated"?
partifipants -> participants
considetare -> considerate
respectfull -> respectfully